calculator/src/test/java/com/market/banica/calculator/componentTests/CalculatorComponentIT.java

```diff
@@ -274,15 +280,36 @@ public void cleanUp() {
     }
 
     @NotNull
-    private RecipeDTO createRecipeDTO() {
-        RecipeDTO response = new RecipeDTO();
+    private ProductDto createProductDTO() {
+        ProductDto response = new ProductDto();
+
+        ProductSpecification productSpecification = createProductSpecification();
```

Comment:
you do not need to create an object, you can directly pass the method where it is used

Reply:
Fixed